LastInfoSec/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  connector-lastinfosec-cti:
+    image: connector-lastinfosec-cti:0.0.5
```

Comment:
The container version should be the current OpenCTI major release `:3.2.0`

Reply:
Fixed :)